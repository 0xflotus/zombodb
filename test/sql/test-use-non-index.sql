-- this should work
SELECT zdb.count('events', 'beer');