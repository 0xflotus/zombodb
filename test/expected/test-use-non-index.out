-- this should work
SELECT zdb.count('events', 'beer');
 count 
-------
    22
(1 row)

